ql/src/test/results/clientpositive/drop_partitions_filter.q.out

```diff
@@ -33,8 +33,20 @@ STAGE DEPENDENCIES:
 STAGE PLANS:
   Stage: Stage-0
     Add Partition
-#### A masked pattern was here ####
-      Spec: {c=US, d=1}
+      db name: default
+      partitions:
+          Partition
+            params:
+              COLUMN_STATS_ACCURATE {"BASIC_STATS":"true","COLUMN_STATS":{"a":"true","b":"true"}}
+              numFiles 0
+              numFilesErasureCoded 0
+              numRows 0
+              rawDataSize 0
+              totalSize 0
+            partition spec:
+              c US
```

Comment:
I think people got used to reading this info in explains like this format; if the partition have multiple keys;
right now it will get split into multiple rows - and will make it harder to search for a specific partition

but anyway: if we would like to change the format how we describe/reference partitions - that should be done as a whole - and not only for addpartition

Reply:
Fixed, sure it looks better the old way.